### PR TITLE
Palindrome Checker in Javascript

### PR DESCRIPTION
**Pull request by tk315**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

Palindrome checker in javaScript done

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#4656

### CODE REVIEW

File: JavaScript/math/Palindrome_checker.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+//To reverse the number
+while(temp!==0)
+{
+    reverse=reverse*10+temp%10;
+    temp=parseInt(temp/10);
+}
+
+if(no===reverse)console.log("Entered Number is a Palindrome");
+else console.log("Entered Number is not a Palindrome");
```
Comment (HarshCasper) [inline JavaScript/math/Palindrome_checker.js]: Keep this inside a function which would be modular

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: JavaScript/math/Palindrome_checker.js (hunk 1 above)
Comment (tk315) [reply JavaScript/math/Palindrome_checker.js]: Done

File: JavaScript/math/Palindrome_checker.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  while (temp !== 0) {
+    reverse = reverse * 10 + (temp % 10);
+    temp = parseInt(temp / 10);
+  }
+
+  if (no === reverse) console.log("Entered Number is a Palindrome");
+  else console.log("Entered Number is not a Palindrome");
+}
+
+palindromechecker(131);
```
Comment (HarshCasper) [inline JavaScript/math/Palindrome_checker.js]: Don't use static input. Use dynamic input

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: JavaScript/math/Palindrome_checker.js (hunk 2 above)
Comment (tk315) [reply JavaScript/math/Palindrome_checker.js]: Done

**Comment by tk315** [comment]
Requested changes done
Kindly review @HarshCasper @ankitaggarwal23 @gat786 @iamrajiv

**Review by iamrajiv — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**